gp-templates/translations.php

```diff
@@ -146,16 +146,14 @@
 			);
 		}
 
-		echo gp_radio_buttons('sort[by]',
-			array(
-				'original_date_added' => __( 'Date added (original)', 'glotpress' ),
-				'translation_date_added' => __( 'Date added (translation)', 'glotpress' ),
-				'original' => __( 'Original string', 'glotpress' ),
-				'translation' => __( 'Translation', 'glotpress' ),
-				'priority' => __( 'Priority', 'glotpress' ),
-				'references' => __( 'Filename in source', 'glotpress' ),
-				'random' => __( 'Random', 'glotpress' ),
-			), gp_array_get( $sort, 'by', $default_sort['by'] ) );
+		$sort_bys = array();
+		$sort_by_fields = gp_get_sort_by_fields();
+
+		foreach ( $sort_by_fields as $key => $field ) {
+			$sort_bys[ $key ] = $field['title'];
+		}
```

Comment:
This can be changed to `$sort_bys = wp_list_pluck( gp_get_sort_by_fields(), 'title' )`.